.github/workflows/ci.yml

```diff
@@ -34,13 +41,10 @@ jobs:
           make
           mkdir firebase
           cp dartLangSpec.pdf firebase/DartLangSpecDraft.pdf
-      - name: Deploy
-        if: |
-          github.event_name == 'push'
-          && github.ref == 'refs/heads/master'
-          && github.repository == 'dart-lang/language'
-        run: |
-          cd specification
-          firebase deploy --project=dart-specification
-        env:
-          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+      - uses: FirebaseExtended/action-hosting-deploy@0e16177d01e10c4560245404f959e75a9c633533
+        with:
+          repoToken: '${{ secrets.GITHUB_TOKEN }}'
+          firebaseServiceAccount: '${{ secrets.FIREBASE_SERVICE_ACCOUNT_PACKAGE_QUALITY }}'
```

Comment:
Looks like this is an empty string when the PR is filed from a fork causing the workflow to fail. That's perhaps an unintended effect?

Reply:
No, I just got the key name wrong!